region/norfair/crocomire/Crocomire's Room.json

```diff
@@ -395,6 +404,27 @@
         {"acidFrames": 55}
       ]
     },
+    {
+      "link": [3, 6],
+      "name": "Acid Bath, Use Flash Suit",
+      "requires": [
+        {"acidFrames": 55},
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 9, "excessFrames": 0}}
+      ],
+      "flashSuitChecked": true,
+      "note": "Jump in the acid multiple times and spark diagonally during the last jump."
+    },
+    {
+      "link": [3, 6],
+      "name": "Long Horizontal Spark, Use Flash Suit",
+      "requires": [
+        {"useFlashSuit": {}},
+        {"shinespark": {"frames": 139, "excessFrames": 108}}
+      ],
+      "flashSuitChecked": true,
+      "note": "Jump and spark horizontally across the room. With a lot of Energy, it would be better to take an acid bath and spark diagonally."
```

Comment:
Shinesparking into the side of the floor and downgrabbing up isn't terribly hard.  Could be added as an insanejump at 32 energy.